packages/docs/core-concepts/index.md

```diff
@@ -27,9 +27,9 @@ Similar to Vue's Options API, we can also pass an Options Object with `state`, `
 
 ```js {2-10}
 export const useCounterStore = defineStore('counter', {
-  state: () => ({ count: 0 }),
+  state: () => ({ count: 0, name: 'Eduardo' }),
```

Comment:
Added the name since we extract it here(line no. 106/110)

`const { name, doubleCount } = store`